gui/vtkMaptkCameraRepresentation.cxx

```diff
@@ -336,8 +336,10 @@ void vtkMaptkCameraRepresentation::Update()
                        this->Internal->ActivePolyData.GetPointer());
   }
 
-  // Path Actor
-  if (this->Internal->PathNeedsUpdate)
+  // Path Actor... only update if we have more than one camera, since we
+  // obviously can't draw lines for a single point, and for some reason having
+  // only one point causes the other actors to fail to render
+  if (this->Internal->PathNeedsUpdate && this->Internal->Cameras.size() > 1)
```

Comment:
I wonder if we still need to 
```
this->Internal->PathPolyData->Reset();
this->Internal->PathPolyData->Modified();
this->Internal->PathNeedsUpdate = false;
```
if there is only one camera

Reply:
It seemed to work, but I didn't test having two cameras and removing one; conceivably that might be broken.